aiida/cmdline/commands/cmd_code.py

```diff
@@ -27,7 +27,24 @@ def verdi_code():
     """Setup and manage codes."""
 
 
-@verdi_code.group('create', cls=DynamicEntryPointCommandGroup, entry_point_name_filter=r'core\.code\..*')
+def create_code(cls, non_interactive, **kwargs):  # pylint: disable=unused-argument
```

Comment:
I am curious is that possible to get rid of using `non_interactive` for these commands?  It is parameters required by `click`, since all these commands I cannot find it being used?

Reply:
I was trying to do this as well, by setting `expose_value=False` on the option itself when declaring it. This way it *is* an option on the command line, but `click` won't pass it to the Python function. However, in this case we actually need it to be passed because the option class `InteractiveOption` relies on it to determine its behavior. If we don't expose it, it thinks it is always interactive and so will always prompt.